### DISCUSSION TITLE
Remove caching from /player/ranked/skywars

### DISCUSSION THREAD

**Discussion by mdashlw** [HypixelDev/PublicAPI#498] [General]
I don't think the API should cache anything, since it consumes the key request limit. 5 whole minutes is a really long time. Also, https://hypixel.net/ does not cache rating.

**Comment by ConnorLinfoot** [comment]
We won't be removing the cache from this endpoint.

**Reply by mdashlw** [reply]
this is very unfortunate, it renders the whole endpoint useless. 5 minutes is unbearable.

**Comment by TheNullicorn** [comment]
As someone who doesn't know much about SkyWars, what's the need for it to refresh more often?
